lib/commander/help_formatters/terminal/command_help.erb

```diff
@@ -14,6 +14,13 @@
 
     <%= Commander::HelpFormatter.indent 4, (@description || @summary || 'No description.') -%>
 
+<% if @explanation -%>
+
+  <%= $terminal.color "EXPLANATION", :bold %>:
+
+    <%= Commander::HelpFormatter.indent 4, (@explanation || 'No explanation.') -%>
```

Comment:
`||` clause is not necessary here since this is within the `if @explanation` check